p2p/p2p.go

```diff
@@ -187,19 +187,24 @@ func (s *Service) Stop() <-chan error {
 }
 
 // Send sends a message to a specific peer
-func (s *Service) Send(peer core.PeerAddrInfo, msg []byte) error {
-	err := s.host.Connect(s.ctx, peer)
-	if err != nil {
-		return err
-	}
+func (s *Service) Send(peer core.PeerAddrInfo, msg []byte) (err error) {
+	log.Info("sending stream", "to", peer.ID, "msg", fmt.Sprintf("0x%x", msg))
 
-	stream, err := s.host.NewStream(s.ctx, peer.ID, protocolPrefix3)
-	if err != nil {
-		return err
+	stream := s.getExistingStream(peer.ID)
+	if stream == nil {
+		stream, err = s.host.NewStream(s.ctx, peer.ID, protocolPrefix2)
+		log.Info("stream", "opening new stream to peer", peer.ID)
```

Comment:
These should not all be `Info`. You have several `Info` level logs for every stream that is opened, closed or used. As I said previously, consider the importance of these relative to the other components of the system. This is not a something that we should change later when we need less verbosity, use `Debug` and `Trace`!

Reply:
ok I will change to debug